composer.json

```diff
@@ -27,22 +27,20 @@
     },
 
     "require": {
-        "php": ">=5.4.0"
+        "php": "^7.1",
+        "ext-json": "*"
     },
 
     "require-dev": {
-        "kriswallsmith/buzz"        : "~0.10",
-        "guzzle/guzzle"             : "~3.7",
-        "guzzlehttp/guzzle"         : "~5.0|~6.0",
-        "phpspec/phpspec"           : "~2.4",
-        "friendsofphp/php-cs-fixer" : "~1.11",
-        "phpdocumentor/phpdocumentor": "2.*"
+        "kriswallsmith/buzz"        : "~0.17",
+        "guzzlehttp/guzzle"         : "~6.0",
+        "phpspec/phpspec"           : "~5.0",
+        "friendsofphp/php-cs-fixer" : "~2.13"
```

Comment:
This won't actually work. They have different config formatting. I'd recommend we actually drop php-cs-fixer from require-dev, and just use https://styleci.io/.

Reply:
I've ripped out php-cs-fixer from master.

Reply:
Have merged upstream so this should be resolved now